### PR TITLE
Cloud ci test and fluvio 0.7 version bump.

### PR DESCRIPTION
**Pull request by simlay**
Needed to make this PR on the infinyon fork because of secrets.

### CODE REVIEW

**Review by sehz — APPROVED** [review]
Nice.  Actually this make it easier to test client

**Comment by simlay** [comment]
I've bundled in the version bump to this PR because I'm lazy.

File: src/consumer.rs [hunk 1]
```diff
@@ -378,7 +378,7 @@ impl<'a> FetchablePartitionResponseWrapper {
 
     #[node_bindgen(getter)]
     fn last_stable_offset(&self) -> Option<i64> {
-        Some(self.0.as_ref()?.last_stable_offset)
+        Some(self.0.as_ref()?.high_watermark)
```
Comment (nicholastmosher) [inline src/consumer.rs:381]: Why is this changing?

Comment (simlay) [reply src/consumer.rs:381]: `last_stable_offset` was set to `deprecated` and I'm lazy.

**Comment by simlay** [comment]
CI for this is an issue with the composite action. Hopefully https://github.com/infinyon/fluvio/issues/1040 leads to way less issues.

**Pull request merged**